src/Brouter/Demo/Bit.Brouter.Demo.Core/Shared/Header.razor.css

```
@@ -0,0 +1,8 @@
+.container {
+    padding: 0.5rem 1rem;
+}
+
+.container ::deep a.active {
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)